config/routes.rb

```diff
@@ -2,15 +2,38 @@
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
+# TASK controller -------------------------------------------
+
   # You can have the root of your site routed with "root"
-  root 'task#index'
+  root 'tasks#index'
+
+  # create a task
+  get  "/tasks/new"           => "tasks#new"
+  post "/tasks/new"           => "tasks#create"
+
+  # read a task
+  get "/tasks/:id/show"       => "tasks#show"
+
+  # update complete field in task
+  patch "/tasks/:id/complete" => "tasks#complete"
+
+  # update task
+  get   "/tasks/:id/edit"     => "tasks#edit"
+  patch "/tasks/:id/edit"     => "tasks#update"
+
+  # delete/destroy task
+  delete "/tasks/delete/:id"  => "tasks#delete"
```

Comment:
It looks like this is the only route that you set up like this - recommend a refactor to /tasks/:id/delete